### DISCUSSION TITLE
Regarding scratch/distroless base image for ppc64le

### DISCUSSION THREAD

**Discussion by sahu-apoorva** [argoproj/argo-cd#17246] [General]
Hi @crenshaw-dev /Adrian Mouat 

I have installed all the dependencies like Apko & Melange to build **wolfi-base** image for ppc64le and tried to build the image by following this document :- https://github.com/chainguard-images/images/blob/main/TERRAFORM.md, but encountered this error :-
````
error getting repository indexes: unable to get repository index at
│ https://packages.wolfi.dev/os/ppc64le/APKINDEX.tar.gz: GET https://packages.wolfi.dev/os/ppc64le/APKINDEX.tar.gz (Range:
│ bytes=0-): unexpected status code: 404
````
It looks like some dependency APK index is not available which is blocking creation of the image for ppc64le. Here I want to understand from you if I am following the correct steps, and please advise me how we can address this issue ?

**Comment by tuananh** [comment]
Wolfi only build for amd64 and arm64 at the moment so theres no apkindex for ppc64le

**Reply by sahu-apoorva** [reply]
Hi @tuananh, 
Thanks for your reply, 
1. I'd like to confirm if the steps I am following to build the **wolfi-base** image are correct.
2. Can you kindly explain how the APKINDEX is generated or where it becomes available?

**Reply by tuananh** [reply]
once you use melange to build package (the apk file), you can use `melange index` to generate the package index.

since Chainguard doesnt build for ppc64le, the index is also not available hence you cannot use it
